### PR TITLE
feat: add 2019-02-07 version

### PR DESCRIPTION
**Pull request by EdouardParis**

### CODE REVIEW

File: content/overview/versions.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ $ curl "https://api.ulule.com/v1/projects/56599" -H Ulule-Version:2017-10-10
 
 ## Versions changelog
 
+### 2019-02-07
+
+* This version changes the [location resource](#location-resource) of a project
```
Comment (yansal) [inline content/overview/versions.md]: can we be more precise? which fields are changing?

Comment (EdouardParis) [reply content/overview/versions.md]: Well, location resource was not documented but I can list the removed fields

Comment (yansal) [reply content/overview/versions.md]: please do it :+1:

File: content/overview/versions.md [hunk 2]
```diff
@@ -21,6 +21,10 @@ $ curl "https://api.ulule.com/v1/projects/56599" -H Ulule-Version:2017-10-10
 
 ## Versions changelog
 
+### 2019-02-07
+
+* This version changes the [location resource](#location-resource) of a project: `name`, `full_name` and `region` are removed.
```
Comment (yansal) [inline content/overview/versions.md]:
```suggestion
* This version removes the `full_name` , `name`, and `region` fields from the [location resource](#location-resource).
```

File: content/reference/project.md [hunk 3]
```diff
@@ -100,6 +100,14 @@ The following payment methods are supported:
 | `paypal`       |
 | `saving`       |
 
+
+### Location resource
+
+| Field     | Type   | Description                        |
+| --------- | ------ | ---------------------------------- |
+| `country` | string | Two-letter ISO code of the country |
```
Comment (yansal) [inline content/reference/project.md]: sort the fields alphabetically

**Pull request merged**